manifests/pin.pp

```diff
@@ -4,7 +4,7 @@
 define apt::pin(
   $ensure          = present,
   $explanation     = undef,
-  $order           = undef,
+  $order           = 50,
```

Comment:
Where does this come from, and why? This affects far more than just the feature you're trying to introduce.